src/tests/mocks/bors.rs

```diff
@@ -293,6 +295,25 @@ impl BorsTester {
         .await
     }
 
+    pub async fn edit_pull_request_with_pr(
+        &mut self,
+        pr_number: u64,
+        pr: GitHubPullRequest,
+    ) -> anyhow::Result<()> {
+        self.send_webhook(
+            "pull_request",
+            GitHubPullRequestEventPayload::new(
+                pr_number,
+                "edited".to_string(),
+                Some(PullRequestChangeEvent {
```

Comment:
We should expose the new base branch in `PullRequestChangeEvent`, and then set it explicitly in this function.